siteapp/views.py

```diff
@@ -1418,6 +1418,59 @@ def new_portfolio(request):
         "project_form": ProjectForm(request.user),
     })
 
+@login_required
+def delete_portfolio(request, pk):
+    """Form to delete portfolios"""
+    if request.method == 'GET':
+        portfolio = Portfolio.objects.get(pk=pk)
+        Portfolio.objects.get(pk=pk).delete()
+
+        logger.info(
+            event="delete_portfolio",
+            object={"object": "portfolio", "id": portfolio.id, "title": portfolio.title},
+            user={"id": request.user.id, "username": request.user.username}
+        )
+        return render(request, "portfolios/index.html", {
+            "portfolios": request.user.portfolio_list() if request.user.is_authenticated else None,
+            "project_form": ProjectForm(request.user),
+        })
+
+@login_required
+def edit_portfolio(request, pk):
+    """Form to create new portfolios"""
+
+    if request.method == 'GET':
+        portfolio = Portfolio.objects.get(pk=pk)
+        form = PortfolioForm(request.POST or None, instance=portfolio, initial={'portfolio': portfolio.id})
+        project_form = ProjectForm(request.user, initial={'portfolio': portfolio.id})
+        if form.is_valid():
+            form.save()
```

Comment:
Since this is still checking for validity, it makes me think the "clean" function should still be present in the form.